appveyor.yml

```diff
@@ -166,10 +166,10 @@ build_script:
   - cd scipy
   - git checkout %BUILD_COMMIT%
   # we use a distribution file to assist in loading
-  # DLLs with recent Python versions (>= 3.8)
+  # DLLs with recent Python versions (>= 3.7)
   - ps: |
       $PYTHON_VERSION = $env:PYTHON_VERSION
-      If ($PYTHON_VERSION -eq 3.8) {
+      If ($PYTHON_VERSION -ge 3.7) {
```

Comment:
Floating-point comparison?

Reply:
Maybe something similar to this to be safe: https://stackoverflow.com/a/55336010/2942522 ?